src/Tracy/Debugger.php

```diff
@@ -400,9 +407,10 @@ public static function setLogger(ILogger $logger)
 	public static function getLogger()
 	{
 		if (!self::$logger) {
-			self::$logger = new Logger(self::$logDirectory, self::$email, self::getBlueScreen());
+			self::$logger = new Logger(self::$logDirectory, self::$email, self::getBlueScreen(), self::$fromEmail);
```

Comment:
That's for BC, I don't see any reason to introduce more properties like this on the Debugger.